Ontology/EPCIS-SHACL.ttl

```diff
@@ -176,488 +168,462 @@ epcis:TransformationEventShape
-    sh:property epcis:CorrectiveEventIDs_TypeConstraint .
-
-
+    sh:property epcis:CorrectiveEventIDs_TypeConstraint
+.
 
 epcis:SourceListShape
     a sh:NodeShape;
-    sh:targetClass epcis:SourceList ;
+    sh:targetObjectsOf epcis:SourceList ;
     sh:property epcis:Source_IsMandatory ;
     sh:property epcis:SourceDestType_IsMandatory ;
     sh:property epcis:Source_TypeConstraint ;
-    sh:property epcis:SourceDestType_TypeConstraint .
-
+    sh:property epcis:SourceDestType_TypeConstraint
+.
 
 epcis:DestinationListShape
     a sh:NodeShape;
-    sh:targetClass epcis:DestinationList ;
+    sh:targetObjectsOf epcis:DestinationList ;
     sh:property epcis:Destination_IsMandatory ;
     sh:property epcis:SourceDestType_IsMandatory ;
     sh:property epcis:Destination_TypeConstraint ;
-    sh:property epcis:SourceDestType_TypeConstraint .
-    
-    
+    sh:property epcis:SourceDestType_TypeConstraint
+.
+
 epcis:QuantityElementShape
     a sh:NodeShape;
-    sh:targetClass epcis:QuantityElement ;
+    sh:targetObjectsOf epcis:QuantityElement ;
     sh:property epcis:EPCClass_IsMandatory ;
     sh:property epcis:Quantity_IsMandatory ;
     sh:property epcis:EPCClass_TypeConstraint ;
     sh:property epcis:Quantity_TypeConstraint ;
-    sh:property epcis:UOM_TypeConstraint .
-    
-    
+    sh:property epcis:UOM_TypeConstraint
+.
 
-
-epcis:Action_IsMandatory owl:sameAs [
+epcis:Action_IsMandatory
     sh:path epcis:action ;
     sh:name "action" ;
     sh:minCount 1 ;
     sh:maxCount 1 ;
-    sh:message "action is a mandatory field which must have a single value." 
-].
+    sh:message "action is a mandatory field which must have a single value."
+.
 
 epcis:Action_TypeAndFormat owl:sameAs [
-    sh:path epcis:action ;
-    sh:datatype xsd:string ;
-    sh:name "action" ;
-    sh:in epcis:ActionEnum ; 
-    sh:message "action must have an upper case string value - either ADD, OBSERVE or DELETE" 
-].
-    
-epcis:ActionEnum owl:sameAs ("ADD" "OBSERVE" "DELETE").
-    
-
-epcis:BizLocation_TypeConstraint owl:sameAs [
+                                          sh:path epcis:action ;
+                                          sh:datatype xsd:string ;
+                                          sh:name "action" ;
+                                          sh:in ("ADD" "OBSERVE" "DELETE") ;
+                                          sh:message "action must have an upper case string value - either ADD, OBSERVE or DELETE"
+                                      ].
+
+epcis:BizLocation_TypeConstraint
     sh:path epcis:bizLocation ;
     sh:nodeKind sh:IRI ;
     sh:name "bizLocation type constraint" ;
-    sh:message "bizLocation must be an xsd:anyURI.  Please refer to the GS1 Core Business Vocabulary for standard vocabulary values for bizLocation" 
-].
-
+    sh:message "bizLocation must be an xsd:anyURI.  Please refer to the GS1 Core Business Vocabulary for standard vocabulary values for bizLocation"
```

Comment:
`xsd:anyURI` is wrong (that's a datatyped literal) and doesn't conform to sh:IRI (which is a URL)